src/components/cards/memberExpanded.js

```diff
@@ -67,6 +68,10 @@ class MemberExpanded extends React.Component {
                         </button>
                       </a>
                     </div>
+                    <div class="text-blue text-right ml-auto w-100 underline">
+                      {" "}
+                      <Link to="/member/">read more</Link>
```

Comment:
remove the last slash, the link should be ` <Link to="/member">read more</Link>`